builtin/game/privileges.lua

```diff
@@ -74,15 +74,15 @@ core.register_privilege("password", {
 	give_to_admin = true,
 })
 core.register_privilege("fly", {
-	description = "Can fly using the free_move mode",
+	description = "Can fly using the fly mode",
```

Comment:
the two words 'fly' seem redundant, though idk how to improve it.

Reply:
It's a reminder that the privilege alone doesn't make you fly directly, but you have to enable the fly mode as well. It's the same writing for `fast` and `noclip`.

Reply:
How about... "Can enable fly mode"?

Reply:
Can use fly mode?

Reply:
What Ezhh suggested.